go/enclave/nodetype/sequencer.go

```diff
@@ -163,6 +163,7 @@ func (s *sequencer) createGenesisBatch(block *common.L1Block) error {
 		return err
 	}
 
+	time.Sleep(time.Second)
```

Comment:
Maybe just add a comment?
`Adds 1 sec delay between Genesis and Batch`

Reply:
Yes ! Thank you, I meant to add that !
```suggestion
        // errors in unit test seem to suggest that batch 2 was received before batch 1
        // this ensures that there is enough gap so that batch 1 is issued before batch 2
	time.Sleep(time.Second)
```